doc/API.md

```diff
@@ -67,16 +67,35 @@ Get an address to build a deposit transaction.
 
 ### Vault resource
 
-| Field         | Type   | Description                                                                          |
-| ------------- | ------ | ------------------------------------------------------------------------------------ |
-| `amount`      | int    | Amount of the vault in satoshis                                                      |
-| `status`      | string | Status of the vault (see [vault statuses](#vault-statuses))                          |
-| `txid`        | string | Deposit txid of the vault deposit transaction                                        |
-| `vout`        | vout   | Index of the deposit output in the deposit transaction.                              |
+| Field         | Type   | Description                                                 |
+| ------------- | ------ | ----------------------------------------------------------- |
+| `amount`      | int    | Amount of the vault in satoshis                             |
+| `blockheight` | int    | Blockheight of the deposit transaction block                |
+| `created_at`  | int    | Timestamp of the deposit transaction block                  |
+| `status`      | string | Status of the vault (see [vault statuses](#vault-statuses)) |
+| `txid`        | string | Deposit txid of the vault deposit transaction               |
+| `updated_at`  | int    | Timestamp of the last status change                         |
+| `vout`        | int    | Index of the deposit output in the deposit transaction.     |
+
+Some fields can be added:
+
+| Field         | Type                                                      | Description                                                                |
+| ------------- | --------------------------------------------------------- | -------------------------------------------------------------------------- |
+| `final_tx`    | [Vault transaction resource](#vault-transaction-resource) | Final transaction -- not present if the vault is not spent or cancelled    |
+| `unvault_tx`  | [Vault transaction resource](#vault-transaction-resource) | Unvault transaction -- not present if vault is not unvaulting or unvaulted |
+
 
 Note that the `scriptPubKey` is implicitly known as we have the vault output Miniscript descriptor.
 **TODO** Maybe we should store and give the xpub derivation index as well ?
 
+### Vault Transaction resource
+
+| Field         | Type   | Description                                      |
+| ------------- | ------ | ------------------------------------------------ |
+| `blockheight` | int    | Block height of the transaction                  |
+| `hex`         | string | Hexadecimal format of the transaction            |
```

Comment:
I choose the hex format of the transaction in order to the client to be able to parse it in case of needs for more tx information

Reply:
Right, so the `txid` is redundant. Also i think the timestamp should be the reception-time rather than block time

Reply:
ACK reception time